cf_remote/spawn.py

```diff
@@ -635,3 +675,151 @@ def dump_vms_info(vms):
             del info[key]
         ret[vm.name] = info
     return ret
+
+
+class VagrantVM(VM):
+
+    def __init__(self, name, ip, vmdir, platform, role, size, cpus, sync_folder):
+        super().__init__(name, role, platform, size, "vagrant", Providers.VAGRANT)
+
+        self.public_ips = [ip]
+        self.region = None
+        self.vmdir = vmdir
+        self.cpus = cpus
+        self.sync_folder = sync_folder
+
+        log.debug(
+            "Created VM with the following information: \n\t- {}\n\t- {}\n\t- {}\n\t- {}".format(
+                name, ip, vmdir, sync_folder
+            )
+        )
+
+    @property
+    def info(self):
+        ret = super().info
+        ret |= {
+            "private_ips": [],
+            "public_ips": self.public_ips,
+            "vmdir": self.vmdir,
+            "cpus": self.cpus,
+            "region": self.region,
+        }
+        if self.sync_folder:
+            ret["sync_folder"] = self.sync_folder
+        elif os.environ.get("NTECH_ROOT"):
+            ret["sync_folder"] = os.environ.get("NTECH_ROOT")
```

Comment:
This environment variable does not seem to match on line 762. Was this intended?